piksi_tools/console/console.py

```diff
@@ -312,7 +317,8 @@ def __init__(self):
 
 with serial_link.get_driver(args.ftdi, port, baud) as driver:
   with sbp.client.handler.Handler(driver.read, driver.write, args.verbose) as link:
-    with serial_link.get_logger(args.log, serial_link.LOG_FILENAME) as logger:
+    logfilename = log_file_prefix + "_" + time.strftime("%Y%m%d-%H%M%S.log.json")
```

Comment:
Or even better, just having the parser argument above be exactly: 

https://github.com/swift-nav/piksi_tools/blob/3fcb4143c0ce3143f1efe311327798873755fdc4/piksi_tools/serial_link.py#L72

so that things will be consistent. Then you can pass in the date, prefix, etc. from the command line.